renovate.json

```diff
@@ -51,6 +51,16 @@
       "depNameTemplate": "com.google.cloud:libraries-bom",
       "datasourceTemplate": "maven"
     },
+    {
+      "fileMatch": [
+        "generation_config.yaml"
+      ],
+      "matchStrings": [
+        "libraries_bom_version\\s*:\\s*(?<currentValue>.*?)\\n"
```

Comment:
Note: I confirmed a similar entry to update gapic_generator_java_version worked via local run of renovate.

Reply:
Thanks.

I also confirm this change works using
```
export LOG_LEVEL=debug && npx --yes --package renovate -- renovate --platform=local
```

Reply:
Thanks guys! What is the process of running renovate locally? I found this [doc](https://docs.renovatebot.com/modules/platform/local/) but not sure if it covers everything.

Reply:
The command can output what renovate will do based on the configuration.

If you checkout this branch and run the command, you will see renovate will update the version in `generation_config.yaml` using regex manager.